src/modules/OpenChannelApp/components/Profile.tsx

```diff
@@ -1,27 +1,28 @@
 import React, { ReactElement } from 'react';
+import { User } from '@sendbird/chat';
 
 import './profile.scss';
 import { ProfileAvatar } from '../assets/Icons';
 
 interface Props {
-  user: SendBird.User;
+  user: User;
 }
 
 export default function Profile({
   user,
 }: Props): ReactElement {
   return (
-    <div className="profile-component">
```

Comment:
Are these intentional changes? double -> single quote

Reply:
It's edited by lint!

Reply:
Oh I thought  "jsx-quotes": ["error", "prefer-double"] has been applied to the project but I guess it's not.